bin/jsondiff.py

```diff
@@ -20,54 +31,48 @@ def add_arguments():
                          help='Specify the maximum relative error (default 1e-8)',
                          type=float,
                          nargs='?', 
-                         const=1e-8)
+                         const=1e-8,
+                         default=None)
     options.add_argument('--abs_tol',
                          metavar='<absolute_error>',
                          dest='absolute_error',
                          help='Specify the maximum absolute error (default 1e-10)',
                          type=float,
                          nargs='?', 
-                         const=1e-10)
+                         const=1e-10,
+                         default=None)
     options.add_argument('-h', '--help', action='help', help='Displays CLI usage statement')
```

Comment:
You don't need this, it is added by default by argparse.
```suggestion
```

Reply:
I made my own help section with the title `Options` (just like the MOOSE help section). To currently have it listed under Options, I needed to set `add_help=False` and add it myself.